client/src/components/App.jsx

```diff
@@ -17,10 +17,14 @@ import {
 } from '@chakra-ui/react';
 import { ChevronDownIcon } from '@chakra-ui/icons';
 import { Link } from "react-router-dom";
+import { searchEventsByTitle } from '../../../server/models/events';
```

Comment:
This line breaks Webpack

Reply:
That actually makes a lot of sense, since it is trying to import node packages into the front end which means it can't find all the node packages.